### PR TITLE
Add batch_external_updates to get_objects2

### PR DESCRIPTION
**Pull request by MrCreosote**
Could significantly speed up fetching many objects with extracted IDs
that need processing, especially once those external systems do a better
job at batching (I'm looking at you blobstore)

Compiled with
```
KBase SDK version 1.2.0 (commit
8def489f648a7ff5657d33ed05f41c60f4766e1b)
```

### CODE REVIEW

File: lib/Bio/KBase/workspace/Client.pm [hunk 1]
```diff
@@ -2543,6 +2543,7 @@ GetObjects2Params is a reference to a hash where the following keys are defined:
 	ignoreErrors has a value which is a Workspace.boolean
```
Comment (MrCreosote) [inline lib/Bio/KBase/workspace/Client.pm:2543]: generated

File: lib/biokbase/workspace/client.py [hunk 2]
```diff
@@ -1525,68 +1525,76 @@ def get_objects2(self, params, context=None):
            Default false. boolean no_data - return the provenance,
```
Comment (MrCreosote) [inline lib/biokbase/workspace/client.py:1525]: generated

File: src/us/kbase/workspace/GetObjects2Params.java [hunk 3]
```diff
@@ -27,10 +27,16 @@
  *                 Default false.
```
Comment (MrCreosote) [inline src/us/kbase/workspace/GetObjects2Params.java:27]: generated

File: src/us/kbase/workspace/kbase/HandleIdHandlerFactory.java [hunk 4]
```diff
@@ -108,29 +109,30 @@ public void addReadPermission(final Collection<String> ids)
...
+					client.addReadAcl(lids, user);
 				}
 			} catch (IOException e) {
 				throw new IdReferencePermissionHandlerException(
 						"There was an IO problem while attempting to set " +
 								"Handle ACLs: " + e.getMessage(), e);
 			} catch (UnauthorizedException e) {
 				throw new IdReferencePermissionHandlerException(
-						"Unable to contact the Handle Manager - " +
+						"Unable to contact the Handle Service - " +
```
Comment (MrCreosote) [inline src/us/kbase/workspace/kbase/HandleIdHandlerFactory.java:125]: The handle manager is pinin' for the fjords

File: src/us/kbase/workspace/test/kbase/HandleAndBytestreamIntegrationTest.java [hunk 5]
```diff
@@ -556,14 +553,106 @@ public void basicHandleTest() throws Exception {
...
+					);
+		} catch (ServerException se) {
+			System.out.println(se.getData());
+			throw se;
+		}
+		
+		CLIENT1.setPermissions(new SetPermissionsParams().withWorkspace(workspace)
+				.withUsers(Arrays.asList(USER2)).withNewPermission("r"));
+		final GetObjects2Params params = new GetObjects2Params()
+				// comment this out to check batching manually, need to add use debugger or
```
Comment (MrCreosote) [inline src/us/kbase/workspace/test/kbase/HandleAndBytestreamIntegrationTest.java:621]: `add` has been removed locally, it'll be fixed whenever I get around to making another WS PR

**Review by ialarmedalien — APPROVED** [review]
Seems reasonable to me.

**Pull request merged**